### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by danielochuba**
# Vet clinic database: create animals table

### In this Pull Request I Completed the following: 
- Created a database named `vet_clinic`.
- Created a table named `animals` with the following columns:
   - `id`: integer
   - `name:` string
   - `date_of_birth`: date
   - `escape_attempts`: integer
   - `neutered:` boolean
   - `weight_kg`: decimal
 - Added the SQL used in the previous point into the `schema.sql` file .
 - Inserted some animal data using the SQL in the ` data.sql` file .
 - Write queries for some animal quests in the `queries.sql` file
 
Screenshot (110)
Screenshot (111)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -256,36 +115,9 @@ If you like this project...
...
-
-## 📝 License <a name="license"></a>
-
-This project is [MIT](./LICENSE) licensed.
+<!-- ACKNOWLEDGEMENTS -->
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
+I would like to thank Microverse for giving me the opportunity to learn and grow as a developer and also I like to thank my family, they are all my support. 🌟
```
Comment (Lucas-Masaba) [inline README.md:119]: - [ ] Please add the `License` header at the end of your README like that in the default [README template](https://github.com/microverseinc/readme-template) provided by Microverse

File: schema.sql [hunk 2]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+id INT,
+name TEXT,
+date_of_birth DATE,
+escape_attempts INT,
+neutered BIT,
```
Comment (Lucas-Masaba) [inline schema.sql]: - [ ] As mentioned in the project requirements, this should be a boolean. It should have either `true` or `false` as their values and not `0` or `1` :+1:

File: schema.sql [hunk 3]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+id INT,
+name TEXT,
```
Comment (Lucas-Masaba) [inline schema.sql]: [OPTIONAL]
- [ ] I suggest you change this from `TEXT` to a `VACHAR()` where you can enter a maximum length of characters to be entered into the brackets.

File: schema.sql [hunk 4]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+id INT,
```
Comment (Lucas-Masaba) [inline schema.sql]: [OPTIONAL]
- You can make the id column not null and autoincremented for example:
`id int NOT NULL IDENTITY(1,1)`

- You can also mark the id column as primary key for example:
`PRIMARY KEY (id)`

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @danielochuba  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost-there

## Highlights 🍾 
- Descriptive pull request message :writing_hand: 
- All SQL is in it's respective file :file_cabinet: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — APPROVED** [review]
## Hi @danielochuba, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Nice job fixing the comments from the previous review :rocket: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by danielochuba** [comment]
@Lucas-Masaba @addod19 thank you all

**Pull request merged**